src/phoenix.js

```diff
@@ -0,0 +1,730 @@
+// Phoenix Channels JavaScript client
```

Comment:
we should include the [npm module instead](https://www.npmjs.com/package/phoenix-js)